distutils/_msvccompiler.py

```diff
@@ -3,8 +3,7 @@
 Contains MSVCCompiler, an implementation of the abstract CCompiler class
 for Microsoft Visual Studio 2015.
 
-The module is compatible with VS 2015 and later. You can find legacy support
-for older versions in distutils.msvc9compiler and distutils.msvccompiler.
+Please ensure that you are using VS 2015 or later.
```

Comment:
nitpick: I try to use third-person imperative voice.

```suggestion
This module requires VS 2015 or later.
```

Reply:
Thank you!